src/js/components/Banner.jsx

```diff
@@ -0,0 +1,26 @@
+import {bannerText} from '../../../config'
+import {grey800, yellow200} from 'material-ui/styles/colors'
+import React, {Component} from 'react'
+
+const style = {
+  banner: {
+    background: grey800,
+    color: yellow200,
+    fontSize: '0.8rem',
+    fontWeight: 400,
+    textAlign: 'center'
+  }
+}
+
+export class Banner extends Component {
```

Comment:
I had banner and footer as pieces inside of App.jsx. You said that they should probably be broken out into component to keep App.jsx small. I think we have a terminology problem, though. There's a difference between a "Component" and a "Functional Component"?